README.md

```diff
@@ -158,6 +158,7 @@ tui-calendar/
 
 * [toast-ui.vue-calendar](https://github.com/nhn/toast-ui.vue-calendar) **Vue** wrapper component is implemented by [NHN](https://github.com/nhn).
 * [toast-ui.react-calendar](https://github.com/nhn/toast-ui.react-calendar) **React** wrapper component is powered by [NHN](https://github.com/nhn).
+* [ngx-mat-tui-calendar](https://github.com/ron2015schmitt/ngx-mat-tui-calendar) **Angular 12** wrapper component [@ron2015schmitt](https://github.com/ron2015schmitt)
```

Comment:
I think you need to mention `Material UI` along with Angular.
And don't forget the `by` preposition while mentioning yourself.